### ISSUE TITLE
Feature Request: Config for JACK ports

### ISSUE THREAD

**Issue by Teteros** [mumble-voip/mumble#3674]
These are configurable at the moment in `~/.config/Mumble/Mumble.conf` for [JACK](https://github.com/mumble-voip/mumble/blob/master/src/mumble/Settings.cpp#L319-L322).
```ini
[jack]
output=1 # or 2 (mono/stereo)
autoconnect=true # or false
startserver=true # or false
```

As is Mumble now connects to the default system ports (likely your dry mic input and speaker/headphoes) but it's very likely you have plugin ports you'd rather have mumble connect to instead.
Maybe a compressor or equalizer at the end of your plugin chain.

Currently it's necessary to do these connections manually or via a `jack_connect` script after mumble starts.

Options for specific `capture` and `playback` ports seem easy to support.

**Comment by gilou** [comment]
Yes, and it would be awesome, is that a goal at some point?

**Comment by gilou** [comment]
also, could we have stereo input?

**Comment by davidebeatrici** [comment]
Yes, we will modify the GUI so that it will expose backend-specific settings.

As for stereo audio transmission, see #2829.

**Comment by Pixdigit** [comment]
As a temporary work-around you can use the qjackctl patchbay to automatically connect to the correct ports.
